algebra/tests/matrix/tools.c

```diff
@@ -202,3 +216,25 @@ int algebraTests_trpCheck(matrix_t *M1, matrix_t *M2)
 
 	return CHECK_OK;
 }
+
+
+int algebraTest_identicalMatrix(matrix_t *M1, matrix_t *M2)
```

Comment:
Shouldn't it be checked correctly by `matrix_cmp`? Would it be possible to test `matrix_cmp` first and then just use it?

Reply:
Ok, thats right. But I found this to be suitable :)
```suggestion
int algebraTest_identicalMatrix(const matrix_t *M1, const matrix_t *M2)
```